parsel/csstranslator.py

```diff
@@ -38,8 +38,8 @@ def __str__(self):
 
         return path
 
-    def join(self, combiner, other):
-        super().join(combiner, other)
+    def join(self, combiner, other, *args, **kwargs):
+        super().join(combiner, other, *args, **kwargs)
```

Comment:
Could you give some context as to why this is now necessary?

Reply:
This class inherits from a cssselect class and the join method there gained two new args in https://github.com/scrapy/cssselect/pull/115

So without this change when you try to use `"a:has(b)"` you get "TypeError: XPathExpr.join() got an unexpected keyword argument 'closing_combiner'"